tools/monitoring/mysql/db_mysql.go

```diff
@@ -118,3 +118,35 @@ func PubsubMsgHandler(db *sql.DB, configURL, buildLogURL, jobname string, prNumb
 
 	return tx.Commit()
 }
+
+// CheckAlertCondition checks whether the given error pattern meets
+// the alert condition specified in config
+func CheckAlertCondition(errorPattern string, config *config.SelectedConfig, db *sql.DB) (bool, error) {
+	// the timestamp we want to start collecting logs
+	startTime := time.Now().Add(time.Minute * time.Duration(config.Period))
+
+	_, err := db.Query(
+		`CREATE VIEW Matched AS
+  SELECT Jobname, PrNumber FROM ErrorLogs
+  WHERE
+        ErrorPattern=? and TimeStamp > ?`,
+		errorPattern, startTime)
```

Comment:
nit: indent or create a const for the sql statements and keep gocode in one line. 
```
_, err := db.Query(
		`CREATE VIEW Matched
		  AS
		     SELECT Jobname, PrNumber
		     FROM ErrorLogs
		     WHERE ErrorPattern = ? and TimeStamp > ?`,
		errorPattern, 
                 startTime)
```

Reply:
my CREATE VIEW Matched AS use the same format as "https://www.w3schools.com/sql/sql_view.asp"

I can change the indent for where clause a bit